chef/cookbooks/crowbar/templates/default/crowbar-monasca-cleanups.sh.erb

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# This script performs various Monasca cleanups required before
+# the Cloud 8 to Cloud 9 upgrade.
+#
+# The script should be run only on the monasca-server node.
+
+LOGFILE=/var/log/crowbar/node-upgrade.log
+UPGRADEDIR=/var/lib/crowbar/upgrade
+STATEFILE=${UPGRADEDIR}/crowbar-remove-java-persister-ok
```

Comment:
nit: `crowbar-remove-monasca-java-persister-ok` to know what this is about.

Reply:
Done (renamed to crowbar-monasca-cleanups-ok since there's more than just the persister now).